src/linear_algebra/bilinear_form.lean

```diff
@@ -351,117 +391,322 @@ end
 
 end
 
+section is_basis
+
+variables {B₃ F₃ : bilin_form R₃ M₃}
+variables {ι : Type*} {b : ι → M₃} (hb : is_basis R₃ b)
+
+/-- Two bilinear forms are equal when they are equal on all basis vectors. -/
+lemma ext_basis (h : ∀ i j, B₃ (b i) (b j) = F₃ (b i) (b j)) : B₃ = F₃ :=
```

Comment:
Should this be `@[ext]`?

Reply:
I don't think that it would work with the `ext` tactic, since it needs a choice of basis `hb`. And there already is an `@[ext] lemma ext : ∀ x y, B x y = F x y -> B = F`.